Modules/itertoolsmodule.c

```diff
@@ -3535,7 +3535,7 @@ static PyMethodDef accumulate_methods[] = {
 };
 
 PyDoc_STRVAR(accumulate_doc,
-"accumulate(iterable[, func]) --> accumulate object\n\
+"accumulate(iterable, func=None) --> accumulate object\n\
```

Comment:
I would like to leave this as-is.   Accepting "None" is an implementation detail that wasn't an intended part of the API, isn't documented, and isn't tested.